Fixed typos

dscheffy opened the pull request.

freastro commented on security/Authentication.rst:
```diff
@@ -24,9 +24,9 @@ for that user, and these groups can have permissions granted to them in Kylo.
-configuration `application.properties` files.  By default, whenever any of these profiles
+Kylo comes with some pre-built authentication configurations that may be
+activated by adding the appropriate Spring profiles to the UI and server
+configuration `application.properties` file.  By default, whenever any of these profiles
```
This should be 'files' because there is more than one (both kylo-ui and kylo-services).

freastro commented on security/Authentication.rst:
```diff
@@ -218,7 +218,7 @@ the only one able to login to Kylo.  Obviously, this profile should only be used
 | Property                       | Required | Example Value | Description                       |
 +================================+==========+===============+===================================+
-| authenticationService.username | Yes      | ``dlamin``    | The username of the administrator |
+| authenticationService.username | Yes      | ``dladmin``    | The username of the administrator |
```
Well this is embarrassing. Thanks for catching this! Could you fix the extra spacing so bars | line up?

dscheffy commented:
I hope you don't mind, but I squashed all of my changes into one commit -- it will look cleaner in the history, but it does make some of your previous comments harder to follow. I changed "file" back to "files" like you asked (I'm going to follow up with a separate question about why we need to make the same changes to both service configurations, but that's a separate issue).  While I was making the change I caught a few other typos and fixed them.
